### DISCUSSION TITLE
Can't download anything

### DISCUSSION THREAD

**Discussion by DinoVikingo** [anidl/multi-downloader-nx#140] [Q&A] [answered]
Everytime i try to download something it comes to this dead end (403: Forbidden)
![image](https://user-images.githubusercontent.com/89960654/140430076-8ce8b88e-27fa-4c10-a497-7c2c49e0f762.png)
In this case i tried with this command:
.\anidl.exe --service "funi" -s 103522 -e 1 -q 0 -allDub --allSubs --fileName "${showTitle} - S${season}E${episode} - ${title}

**Comment by izu-co** [comment, marked as answer]
If you think you've spotted an error please open up an issue instead of a discussion.

Thank you :)
